src/serializers/type_serializers/function.rs

```diff
@@ -185,6 +191,7 @@ pub struct FunctionWrapSerializer {
     function_name: String,
     json_return_ob_type: Option<ObType>,
     when_used: WhenUsed,
+    needs_model_instance: bool,
```

Comment:
I think a bool is fine here, probably worth minimizing boilerplate for now considering this is nascent enough that it might still get significant changes for reasons we haven't thought of yet.